build-logic/src/main/kotlin/buildlogic.kotlin-publishing-conventions.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    id("org.jetbrains.dokka")
+    signing
+    `maven-publish`
+}
+
+// Javadoc is mandatory on maven central
+val dokkaHtml by tasks.getting(DokkaTask::class)
+dokkaHtml.dependsOn(":csaf-schema:generate")
+val javadocJar by tasks.registering(Jar::class) {
+    dependsOn(dokkaHtml)
+    archiveClassifier.set("javadoc")
+    from(dokkaHtml.outputDirectory)
+}
+
+// Publication settings for maven central
+publishing {
+    publications {
+        create<MavenPublication>(name) {
+            artifact(javadocJar)
+            from(components["java"])
+
+            pom {
+                url.set("https://github.com/csaf-sbom/kotlin-csaf")
+                licenses {
+                    license {
+                        name.set("The Apache License, Version 2.0")
+                        url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+                developers {
+                    developer {
+                        id.set("oxisto")
```

Comment:
Ist das Absicht, dass du dich hier verewigen möchtest? :wink:

Reply:
As far as I know, at least one developer ID needs to be set; otherwise I had problems that the POM was rejected by maven central